server/index.js

```diff
@@ -10,28 +10,75 @@ app.use(express.static(`${__dirname}/../client/public`));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
-// ROUTES
-app.get('/api/reviews/:search', (req, res) => {
-  db.find(req.params.search, (err, results) => {
+// OLD FRONT END ROUTES
+app.patch('/api/reviews/:id', ({ params: { id }, body: { reviewId, field, value } }, res) => {
+  db.update(id, reviewId, field, value, (err, results) => {
     if (err) {
-      res.status(400).send(err);
+      res.sendStatus(400);
     } else {
```

Comment:
I remember Robin saying, "don't ever send errors from the back end to the client" or something like that. Errors can contain too much information. So good job changing that!

Reply:
thanks @jayrbarr !